File: exam-recap-ui/exam-recap-ui/src/app/data/game.ts
export class Game {
  id: number;
  userId: number;
  score: number;
  draws: number;
  status: string = "";

  constructor(id: number, userId: number, score: number, draws: number, status: string) {
    this.id = id;
    this.userId = userId;
    this.score = score;
    this.draws = draws;
    this.status = status;
  }
}

File: exam-recap/exam-recap/src/main/java/org/example/examrecap/model/Game.java
package org.example.examrecap.model;

import jakarta.persistence.*;
import lombok.*;
import java.time.LocalDateTime;

@Entity
@Table(name = "games")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Game {
  @Column
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private long id;
  @Column
  private long userId;
  @Column
  private long score;
  @Column
  private long draws;
  @Column
  @Enumerated(EnumType.STRING)
  private GameStatus status;
  @Column
  private LocalDateTime timestamp;
}